lms/djangoapps/courseware/access_utils.py

```diff
@@ -75,6 +76,7 @@ def check_start_date(user, days_early_for_beta, start, course_key, display_error
     Returns:
         AccessResponse: Either ACCESS_GRANTED or StartDateError.
     """
+    from openedx.features.enterprise_support.api import enterprise_customer_from_session_or_learner_data
```

Comment:
Don't really understand what's going on here, but adding this at the top of the file caused a circular import error.  This pattern is used elsewhere in this same file so I just copied it.